### PR TITLE
Fix permission test

### PR DESCRIPTION
**Pull request by yaziine**

### CODE REVIEW

File: permission_client_test.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func TestPermissions_PermissionEndpoints(t *testing.T) {
 	perm := resp.Permission
 	assert.Equal(t, "create-channel", perm.ID)
 	assert.False(t, perm.Custom)
-	assert.NotEmpty(t, perm.Condition)
+	assert.Empty(t, perm.Condition)
```
[the remark this answers is not in the document]
Comment (AnatolyRugalev) [reply permission_client_test.go:66]: I think it's fine to keep this PR as is and if we really need to, add custom permission testing later. I don't see a lot of value in these tests today tbh. The purpose of the PR is to fix CI

**Review by AnatolyRugalev — APPROVED** [review]

File: permission_client_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (ferhatelmas) [reply permission_client_test.go:66]: how do we know it should be empty ? IMHO it should be checked so only if custom is false

[the remark this answers is not in the document]
Comment (yaziine) [reply permission_client_test.go:66]: we are already checking that it's not a custom permission line 65 so i think it's enough, isn't it?

[the remark this answers is not in the document]
Comment (ferhatelmas) [reply permission_client_test.go:66]: Yup, good to go

**Pull request merged**